azure-functions-codegen/src/func.rs

```diff
@@ -83,6 +85,44 @@ fn validate_orchestration_function(func: &ItemFn) {
     }
 }
 
+fn validate_activity_function(func: &ItemFn) {
+    // Activity functions cannot have a $return binding
+    // Default, -> ActivityOutput, and -> (ActivityOutput, ...) are acceptable
+
+    fn validate_return_binding(ty: &Type) {
```

Comment:
I assume `type` is reserved in Rust otherwise you'd use it?

Reply:
That's correct.